### ISSUE TITLE
Crash after sleeping

### ISSUE THREAD

**Issue by jtmorphew** [anegostudios/VintageStory-Issues#2102]
**Game Version:** 1.17 rc4
**Platform:** Windows /  
**Modded:** No /  
**SP/MP:** Singleplayer 

### Description
Crash after sleeping in bed near bee hive... unsure if the two are related.

### How to reproduce
see above

### Logs
```
Running on 64 bit Windows with 32 GB RAM 
Game Version: v1.17.0-rc.4 (Unstable)
Loaded Mods: game@1.17.0-rc.4, creative@1.17.0-rc.4, survival@1.17.0-rc.4
8/9/2022 10:35:27 AM: Critical error occurred
System.NullReferenceException: Object reference not set to an instance of an object.
   at Vintagestory.GameContent.BlockEntityFruitTreePart.GenFoliageMesh(Boolean withSticks, MeshData& foliageMesh, MeshData& sticksMesh) in C:\Users\Tyron\Documents\vintagestory\game\VSSurvivalMod\Systems\FruitTree\BEFruitTreePart.cs:line 209
   at Vintagestory.GameContent.BlockEntityFruitTreeBranch.GenMeshes() in C:\Users\Tyron\Documents\vintagestory\game\VSSurvivalMod\Systems\FruitTree\BEFruitTreeBranch.cs:line 241
   at Vintagestory.GameContent.BlockEntityFruitTreeBranch.GenMesh() in C:\Users\Tyron\Documents\vintagestory\game\VSSurvivalMod\Systems\FruitTree\BEFruitTreeBranch.cs:line 228
   at Vintagestory.GameContent.BlockEntityFruitTreeBranch.FromTreeAttributes(ITreeAttribute tree, IWorldAccessor worldForResolving) in C:\Users\Tyron\Documents\vintagestory\game\VSSurvivalMod\Systems\FruitTree\BEFruitTreeBranch.cs:line 372
   at Vintagestory.Client.NoObf.ClientChunk.AddOrUpdateBlockEntityFromPacket(Packet_BlockEntity p, ClientMain game) in C:\Users\Tyron\Documents\vintagestory\game\VintagestoryLib\Client\Model\ClientChunk.cs:line 409
   at Vintagestory.Client.NoObf.GeneralPacketHandler.HandleBlockEntities(Packet_Server packet) in C:\Users\Tyron\Documents\vintagestory\game\VintagestoryLib\Client\Systems\GeneralPacketHandler.cs:line 808
   at Vintagestory.Client.NoObf.ClientMain.ExecuteMainThreadTasks(Single deltaTime) in C:\Users\Tyron\Documents\vintagestory\game\VintagestoryLib\Client\ClientMain.cs:line 1138
   at Vintagestory.Client.GuiScreenRunningGame.RenderToPrimary(Single dt) in C:\Users\Tyron\Documents\vintagestory\game\VintagestoryLib\Client\MainMenu\Screens\GuiScreenRunningGame.cs:line 123
   at Vintagestory.Client.ScreenManager.Render(Single dt) in C:\Users\Tyron\Documents\vintagestory\game\VintagestoryLib\Client\ScreenManager.cs:line 664
   at Vintagestory.Client.ScreenManager.OnNewFrame(Single dt) in C:\Users\Tyron\Documents\vintagestory\game\VintagestoryLib\Client\ScreenManager.cs:line 608
   at Vintagestory.Client.NoObf.ClientPlatformWindows.window_RenderFrame(Object sender, FrameEventArgs e) in C:\Users\Tyron\Documents\vintagestory\game\VintagestoryLib\Client\ClientPlatform\GameWindow.cs:line 125
   at System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   at OpenTK.GameWindow.RaiseRenderFrame(Double elapsed, Double& timestamp) in C:\Users\Nexrem\Desktop\transfer\opentk\src\OpenTK\GameWindow.cs:line 476
   at OpenTK.GameWindow.DispatchRenderFrame() in C:\Users\Nexrem\Desktop\transfer\opentk\src\OpenTK\GameWindow.cs:line 452
   at OpenTK.GameWindow.Run(Double updates_per_second, Double frames_per_second) in C:\Users\Nexrem\Desktop\transfer\opentk\src\OpenTK\GameWindow.cs:line 375
   at Vintagestory.Client.ClientProgram.Start(ClientProgramArgs args, String[] rawArgs)
   at Vintagestory.ClientNative.CrashReporter.Start(ThreadStart start) in C:\Users\Tyron\Documents\vintagestory\game\VintagestoryLib\Client\ClientPlatform\ClientNative\CrashReporter.cs:line 87
-------------------------------

Event Log entries containing Vintagestory.exe, the latest 3
==================================
{ TimeGenerated = 8/9/2022 10:28:47 AM, Site = , Source = Windows Error Reporting, Message = Fault bucket 1269912095728824642, type 4
Event Name: APPCRASH
Response: Not available
Cab Id: 0

Problem signature:
P1: Vintagestory.exe
P2: 1.17.0.0
P3: 62ef7eff
P4: KERNELBASE.dll
P5: 10.0.19041.1826
P6: 299341e8
P7: c0020001
P8: 0000000000034fd9
P9: 
P10: 

Attached files:
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER7D85.tmp.WERInternalMetadata.xml

These files may be available here:
\\?\C:\ProgramData\Microsoft\Windows\WER\ReportArchive\AppCrash_Vintagestory.exe_c3413cf49ae7fd54f941f2f16f196a643a5337ad_89e9a2e0_5a6ecc99-f51e-4f66-88b1-f37814cf893f

Analysis symbol: 
Rechecking for solution: 0
Report Id: 9be508fc-6fec-44d7-a211-5ab4f6820e70
Report Status: 268566528
Hashed bucket: feb1413d797df67e819fa252a1e37142
Cab Guid: 0 }
--------------
{ TimeGenerated = 8/9/2022 10:28:46 AM, Site = , Source = Windows Error Reporting, Message = Fault bucket , type 0
Event Name: APPCRASH
Response: Not available
Cab Id: 0

Problem signature:
P1: Vintagestory.exe
P2: 1.17.0.0
P3: 62ef7eff
P4: KERNELBASE.dll
P5: 10.0.19041.1826
P6: 299341e8
P7: c0020001
P8: 0000000000034fd9
P9: 
P10: 

Attached files:

These files may be available here:
\\?\C:\ProgramData\Microsoft\Windows\WER\ReportQueue\AppCrash_Vintagestory.exe_c3413cf49ae7fd54f941f2f16f196a643a5337ad_89e9a2e0_5a6ecc99-f51e-4f66-88b1-f37814cf893f

Analysis symbol: 
Rechecking for solution: 0
Report Id: 9be508fc-6fec-44d7-a211-5ab4f6820e70
Report Status: 131076
Hashed bucket: 
Cab Guid: 0 }
--------------
{ TimeGenerated = 8/9/2022 10:28:41 AM, Site = , Source = Application Error, Message = Faulting application name: Vintagestory.exe, version: 1.17.0.0, time stamp: 0x62ef7eff
Faulting module name: KERNELBASE.dll, version: 10.0.19041.1826, time stamp: 0x299341e8
Exception code: 0xc0020001
Fault offset: 0x0000000000034fd9
Faulting process id: 0x3ac
Faulting application start time: 0x01d8abfba30f643f
Faulting application path: E:\Program Files (x86)\Vintagestory\Vintagestory.exe
Faulting module path: C:\WINDOWS\System32\KERNELBASE.dll
Report Id: 9be508fc-6fec-44d7-a211-5ab4f6820e70
Faulting package full name: 
Faulting package-relative application ID:  }

**Comment by Craluminum2413** [comment]
[This issue was addressed in 1.18.0-pre.8](https://info.vintagestory.at/v1dot18) as `Fixed: Should avoid a crash related to fruit trees`,

**Comment by Craluminum2413** [comment]
If the same error happen with anyone in `1.18.0-pre.8` or higher, I will reopen it
